tests/test_decorator_no_args.py

```diff
@@ -1,51 +1,35 @@
-from datetime import datetime
-from typing import Tuple
-from unittest.mock import MagicMock
+from typing import List, Tuple
 
 import pytest
-from pytest_mock import MockerFixture
+from boto3 import Session
+from mypy_boto3_organizations.type_defs import AccountTypeDef
 
 from botocove import CoveSession, cove
 
 
 @pytest.fixture()
-def patch_boto3_client(mocker: MockerFixture) -> MagicMock:
-    mock_boto3 = mocker.patch("botocove.cove_host_account.boto3")
-    list_accounts_result = {"Accounts": [{"Id": "12345689012", "Status": "ACTIVE"}]}
-    mock_boto3.client.return_value.get_paginator.return_value.paginate.return_value.build_full_result.return_value = (  # noqa E501
-        list_accounts_result
-    )
-    describe_account_results = [
-        {
-            "Account": {
-                "Id": "1234",
-                "Arn": "hello-arn",
-                "Email": "email@address.com",
-                "Name": "an-account-name",
-                "Status": "ACTIVE",
-                "JoinedMethod": "CREATED",
-                "JoinedTimestamp": datetime(2015, 1, 1),
-            }
-        }
-    ]
-    mock_boto3.client.return_value.describe_account.side_effect = (
-        describe_account_results
-    )
-    return mock_boto3
+def org_accounts(mock_session: Session) -> List[AccountTypeDef]:
```

Comment:
Do we need to repeat this in every test?

Reply:
I wonder if it's worth just making this a BarebonesOrg() class like the other sizes

Reply:
I wasn't sure about the right way to abstract it so I didn't try to deduplicate it. Some `org_accounts` fixtures create one member account and others create two. The reason for one size or another is typically embedded in comments of the tests they are next to.

Maybe instead of a one-size-fits all BareBonesOrg, we could have a parametrized fixture that creates a flat org of N member accounts. N would be specified as 1 or 2 in each test.

Reply:
ah, I missed that on my skim through. Makes sense, it's fine to just keep this way - honestly my plan was to just delete the legacy tests at some point so I appreciate this effort :) Happy with your steer on that, I don't want to push extra busy work in a test harness that's pretty light on repetition.

Reply:
I'll leave it as it is for now then because I'm keen to get on to the issues for adding new features. :-) We can always refactor the test code again in another PR if it helps us to understand things better.